packages/swing-store/src/snapStore.js

```diff
@@ -142,8 +181,8 @@ export function makeSnapStore(
 
   const sqlSaveSnapshot = db.prepare(`
     INSERT OR REPLACE INTO snapshots
-      (vatID, endPos, inUse, hash, uncompressedSize, compressedSize, compressedSnapshot)
-    VALUES (?, ?, 1, ?, ?, ?, ?)
+      (vatID, endPos, hash, uncompressedSize, compressedSize, compressedSnapshot, inUse)
```

Comment:
Sqlite _does_ have a way to provide non-positional parameters to prepared statements; it's just notationally a lot more awkward so I haven't used it.  But making the orders of the function and the prepared statement match is straightforward.  I did it the way I did because from my perspective at the time I was adding a new parameter, so tacked it onto the end as the least disruptive change.  But having the orders match is cleaner.

On the other hand, I don't think there's a lot of point to matching against the `CREATE TABLE` column order, as most queries only use some of the columns and it makes the queries clearer if they order things according to the importance of what's being referenced.  Also,`CREATE TABLE` lays out the columns vertically whereas queries are horizontal, so matching them up doesn't come naturally anyway, especially if there are gaps (which there almost always are).